docs/source/user/utilities/typing.rst

```diff
@@ -0,0 +1,123 @@
+Typing
+======
+
+OpenDP computations are always strict about the types being used. 
+Integers and floats are never treated interchangeably and there is never implicit casting between types.
+In fact, OpenDP is particular about the bit-depth of data types, as it can impact the choice of constants in the privacy analysis.
+
+.. _RuntimeTypeDescriptor:
+
+Type Argument
+-------------
+
+You can explicitly set the types that a transformation or measurement work with via type arguments in the constructor.
+Each constructor has its own set of permissible types, based on the type of computation it is performing.
+For instance, the clamp constructor accepts any numerical type for its type argument `TA`:
+
+.. testsetup::
+
+    from opendp.mod import enable_features
+    enable_features('contrib')
+
+.. doctest::
+
+    >>> from opendp.transformations import make_clamp
+    >>> from opendp.typing import *
+    ...
+    >>> clamper = make_clamp((0, 1), TA=i32)
+
+Many of the API docs indicate that parameters like `TIA` or `D` are type arguments.
+When you want to describe the type of a domain, metric, measure, or other elements, you can do so via a type descriptor.
+The canonical form is a literal string, like `"i32"` to denote that the type should be a 32-bit integer,
+or `"SymmetricDistance"` to denote the type of a metric.
```

Comment:
The straight quotes around the type names get converted by sphinx to curly quotes, but here I think it'd be better for them to stay as straight quotes?